### ISSUE TITLE
[BUG]: Getting error - "Cannot find module './genReport'" 

### ISSUE THREAD

**Issue by caioafc** [IBMa/equal-access#287]
## What tool(s) are you using?

- [ ] `accessibility-checker-engine`
- [ ] `accessibility-checker`
- [ ] `karma-accessibility-checker`
- [ ] `rule-server`
- [ ] `boilerplates`
- [ ] `other`
- [x] `cypress-accessibility-checker`

## Detailed description

> Node version: v12.16.1
> Platform: Windows 10 Enterprise

We are getting "Error: Cannot find module './genReport'" when runs cypress project with cypress-accessibility-checker package configured, stack trace:

C:\CIO\cypress-automated-test\BP-Portal-Refactor\node_modules\cypress-accessibility-checker\src\lib\reporters\ACReporterHTML.js
C:\CIO\cypress-automated-test\BP-Portal-Refactor\node_modules\cypress-accessibility-checker\src\lib\ACTasks.js
C:\CIO\cypress-automated-test\BP-Portal-Refactor\node_modules\cypress-accessibility-checker\src\tasks.js
C:\CIO\cypress-automated-test\BP-Portal-Refactor\node_modules\cypress-accessibility-checker\plugin.js
C:\CIO\cypress-automated-test\BP-Portal-Refactor\cypress\plugins\index.js
 C:\Users\CaioArmestrongFernan\AppData\Local\Cypress\Cache\6.0.0\Cypress\resources\app\packages\server\lib\plugins\child\run_plugins.js
 C:\Users\CaioArmestrongFernan\AppData\Local\Cypress\Cache\6.0.0\Cypress\resources\app\packages\server\lib\plugins\child\index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:972:15)
    at Module._load (internal/modules/cjs/loader.js:848:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12694)
    at Module.require (internal/modules/cjs/loader.js:1032:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\CIO\cypress-automated-test\BP-Portal-Refactor\node_modules\cypress-accessibility-checker\src\lib\reporters\ACReporterHTML.js:21:23)
    at Module._compile (internal/modules/cjs/loader.js:1152:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1173:10)
    at Module.load (internal/modules/cjs/loader.js:992:32)
    at Module._load (internal/modules/cjs/loader.js:885:14)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12694)
    at Module.require (internal/modules/cjs/loader.js:1032:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\CIO\cypress-automated-test\BP-Portal-Refactor\node_modules\cypress-accessibility-checker\src\lib\ACTasks.js:4:24)
    at Module._compile (internal/modules/cjs/loader.js:1152:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1173:10)

## Steps to reproduce the issue

1. Install accessbility checker dependency --> npm install cypress-accessibility-checker --save-dev
2. Setup cypress plugin configuration at plugins/index.js file
3. Create and setup .achecker.yml file at node_modules/cypress-accessibility-checker directory
4. Run cypress project

## Additional information
Error:
![image](https://user-images.githubusercontent.com/52216605/103026696-36cd3980-4533-11eb-96f9-a969eb8d57e2.png)

plugins/index.js file:
![image](https://user-images.githubusercontent.com/52216605/103026742-4fd5ea80-4533-11eb-9768-f35333ab8df3.png)

The genReport file doesn't exists:
![image](https://user-images.githubusercontent.com/52216605/103027238-38e3c800-4534-11eb-9397-2559baa9b5f7.png)

**Comment by tombrunet** [comment]
Did a test build and the file appears to be there. Will have to check after the next release - hoping to try this week.

**Comment by tombrunet** [comment]
The file is in the package directory when we publish, but it's not showing up on NPM. I will continue to hunt, but it's not obvious why this particular file is not being included.

**Comment by tombrunet** [comment]
Found it..  there was a gitignore file in the package for development, but npm was seeing that and ignoring the file as part of the package. I'll get this pushed shortly. https://github.com/IBMa/equal-access/pull/318
